web/mailer.ex

```diff
@@ -3,9 +3,6 @@ defmodule HexWeb.Mailer do
     assigns = [layout: {HexWeb.EmailView, "layout.html"}] ++ assigns
     body    = Phoenix.View.render(HexWeb.EmailView, template, assigns)
 
-    HexWeb.Utils.task(fn ->
-      fun = fn email -> HexWeb.Email.send([email], title, body) end
-      HexWeb.Parallel.run(fun, emails)
-    end)
+    HexWeb.Email.send(emails, title, body)
```

Comment:
Good point, I fixed it in c0b0f76. I figured to put that logic in Email.SES since it depends on SES settings